### ISSUE TITLE
[A11Y] The search result, or filtering result, is not announced to Assitive Technology users, e.g. Scrennreaders on the Submissions Dashboard

### ISSUE THREAD

**Issue by Devika008** [pkp/pkp-lib#9359]
## Issue Description
// What is the nature of the issue? How does it impact users?

Using the search field on the submissions panel, it is possible to filter the results. However, after typing terms to filter, the list changes but the results is not announced. The user will not be aware of any change on the listing and the experience might become frustrating.

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question

1. Go on the main submissions panel
2. In the search field, try searching for submissions via the author name 
3. View the results 

## What application are you using?
 OJS, OMP or OPS version X.X.X
 
 OJS 3.4 

## Stack used 

- Device: laptop
- OS: MacOS (Monterey)
- Browser: Chrome
- Version: 117

## Testing method
- Screen Reader (VoiceOver)

## Action
// Describe action that can be taken to remediate the issue. 

- Add role=status to the panel that shows the filtered results.
- Will show it as a UI element on the new submission dashboard as well to indicate to all users

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

![image](https://github.com/pkp/pkp-lib/assets/112014397/89a774f5-d8fd-4a9e-97cc-149ab7a9c6c4)

**Comment by Devika008** [comment]
In 3.4, I propose to include the following text as a UI element below Headline **unassigned**: We found 6 results that match your search criteria

As part of the 3.5 redesign of Submission Lists, I have included the following text as a UI signal to the users:

![image](https://github.com/pkp/pkp-lib/assets/112014397/19873159-a292-40d7-ad9c-15e4c098778e)

Also @israelcefrin further reiterates that, maybe we should map all "living regions" of the interface that might need this role. I assume that different journeys (reviewing, inviting collaborators) could harness from accessible living regions.
